Art/ahmedraza.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
+        * {
+  margin: 0;
+  padding: 0;
+}
+body {
+  background: #252525;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.dot-loader {
+  display: inline-block;
+  margin-top: 25px;
+}
+.dot-loader .dot {
+  animation: dot 1100ms forwards infinite;
+  background: #0cffff;
+  border-radius: 50%;
+  box-shadow: 0 0 8px #0abab5;
+  display: inline-block;
+  height: 10px;
+  margin: 6px;
+  transform: scale(0, 0);
+  width: 10px;
+}
+.dot-loader .dot--2,
+.dot-loader .dot--4 {
+  animation-delay: 125ms;
+}
+.dot-loader .dot--1,
+.dot-loader .dot--5 {
+  animation-delay: 250ms;
+}
+
+@keyframes dot {
+  0% {
+    transform: scale(0, 0);
+  }
+  12% {
+    transform: scale(1.5, 1.5);
+  }
+  23% {
+    transform: scale(0.5, 0.5);
+  }
+  34% {
+    transform: scale(1.2, 1.2);
+  }
+  45% {
+    transform: scale(1, 1);
+  }
+  85% {
+    transform: scale(0, 0);
+  }
+}
+
+    </style>
```

Comment:
# Feedback - modularisations
Your current code contains both HTML and CSS whereas the guidelines encourage
to use modularisations ( separating HTML and CSS in order to import the file )
- [ ] modularise your CSS into 
   make a file CSS dissociated from your HTML, and import this new CSS file
   into your HTML file.
   Here a resource for you to understand how it is done : [w3schools - html link](https://www.w3schools.com/tags/tag_link.asp)